pkg/catalog/configmeta.go

```diff
@@ -182,11 +297,21 @@ func clusterBaseConfig(
 	appCR *kdv1.KubeDirectorApp,
 	membersForRole map[string][]*kdv1.MemberStatus,
 	domain string,
-) *configmeta {
+) (*configmeta, error) {
+
+	clustersMeta, connErr := genClusterConnections(cr)
+	kdConfigMaps, cmErr := genconfigConnections(cr)
```

Comment:
Similar to my comments above about the CR property... I think we can take this opportunity, if we want to, to improve the JSON representation of these things. I'm not familiar enough with that though to know what we might need to do.

Reply:
But e.g. we don't need to and probably shouldn't stick them in a wrapper "Connections" namespace together.